kong/runloop/handler.lua

```diff
@@ -78,13 +78,26 @@ do
     return false
   end
 
+
+  local function load_service_from_db(service_pk)
+    return kong.db.services:select(service_pk)
```

Comment:
Better discard the third return value here, which is interpreted as an overridden TTL by mlcache. mlcache plays safe and will discard it if not a number, but let's be explicit about this and not rely on such low-level behavior.